server/sonar-web/src/main/js/apps/projects/components/ProjectOptionBar.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+//@flow
+import React from 'react';
+import classNames from 'classnames';
+import PerspectiveSelect from './PerspectiveSelect';
+import { translate } from '../../../helpers/l10n';
+
+type Props = {
+  onPerspectiveChange: ({ view: string, visualization?: string }) => void,
+  onToggleOptionBar: boolean => void,
+  open: boolean,
+  view: string,
+  visualization?: string
+};
+
+export default class ProjectOptionBar extends React.PureComponent {
+  props: Props;
+
+  openBar = (evt: Event & { currentTarget: HTMLElement }) => {
+    evt.currentTarget.blur();
+    evt.preventDefault();
+    this.props.onToggleOptionBar(true);
+  };
+
+  closeBar = (evt: Event & { currentTarget: HTMLElement }) => {
+    evt.currentTarget.blur();
+    evt.preventDefault();
+    this.props.onToggleOptionBar(false);
+  };
+
+  render() {
+    const { open } = this.props;
+    return (
+      <div className="projects-topbar">
+        <a className="button projects-topbar-open" href="#" onClick={this.openBar}>
+          {translate('projects.view_settings')}
```

Comment:
Yeah after discussion with Laura we decided to remove it. Mainly because there is nearly nowhere icons in buttons in the application.